### DISCUSSION TITLE
Not able to sign in or signup Vercel

### DISCUSSION THREAD

**Discussion by Yogesh-vaionex** [vercel/community#4677] [Account Issue] [answered]
### Question

![Screenshot from 2023-10-31 17-23-47](https://github.com/vercel/community/assets/149484717/cb29d22e-0f64-4846-b6cf-2fa742e6b6e9)
![Screenshot from 2023-10-31 16-01-51](https://github.com/vercel/community/assets/149484717/61b68c20-c948-4e39-b992-4a156c96706f)
![Screenshot from 2023-10-31 16-13-53](https://github.com/vercel/community/assets/149484717/edf6a8c8-7b31-45ae-8af1-9d09bb94d72d)

I have been having issues while i was trying to commit the changes via github branch. After committing the changes i tried to create a PR but there was an error of vercel "Vercel — Git author must have access to the project on Vercel." The organization's repo wass connected with my personal github and i was able to raise the PR and merge also via it. Later I enabled 2-way auth for github but i dont think thats the problem. 
Later When I tried to raise a PR, i was getting same errors. I revoked the access from everywhere, then tried to log in via github, email, ssl etc. but no success. 
What should I do?

@vercel-infra @github

**Comment by amyegan** [comment, marked as answer]
🙁 I'm sorry you're having trouble logging in. We can't safely discuss account information in public because of security concerns.

To open a ticket, please use the appropriate [direct channel](https://vercel.com/guides/how-to-get-vercel-support#direct-channels) found in this guide: [What is the best way to get support from Vercel?](https://vercel.com/guides/how-to-get-vercel-support#direct-channels)
